modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-item-selector-api/src/main/java/com/liferay/dynamic/data/mapping/form/item/selector/criterion/DDMUserPersonalFolderItemSelectorCriterion.java

```diff
@@ -36,6 +36,10 @@ public long getFolderId() {
 		return _folderId;
 	}
 
+	public long getFormInstanceId() {
+		return _formInstanceId;
+	}
+
```

Comment:
@linolaoj can you rename the methods to `getDDMFormInstanceId` and `setDDMFormInstanceId` and the variable to `_ddmFormInstanceId`? Thanks